TSC-Charter.md

```diff
@@ -162,12 +181,13 @@ for dissent from the consensus.
 If an agenda item cannot reach a consensus a TSC member can call for
 either a closing vote or a vote to table the issue to the next meeting.
 The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue. Simple majority wins, with the following
-exceptions, which will require the affirmative vote of two-thirds of the
-members of the TSC to pass:
+discussion will continue. Simple majority wins with exception given only to
+removal of a member from the TSC, which shall require a two-thirds majority
+vote to approve.
```

Comment:
If anything it makes *less* sense for removal to have a higher bar because most removals happen when it becomes difficult to make quorum, which would mean 66% is near impossible if you're having regular quorum issues.

I've said this many times but we should **1)** try to never resort to votes and when we do have to vote **2)** stick to simple majorities. 

Raising the bar above a simple majority favors a lack of change and progress, makes it more difficult to get things done, and puts an undue burden on those who must convince their peers to enact changes compared to those who do not.